litgpt/deploy/serve.py

```diff
@@ -82,7 +82,7 @@ def setup(self, device: str) -> None:
 
     def decode_request(self, request: Dict[str, Any]) -> Any:
         # Convert the request payload to your model input.
-        prompt = request["prompt"]
+        prompt = str(request["prompt"])
```

Comment:
@rasbt just curious about why did we have to cast the type like this?

Reply:
Good question @aniketmaurya . I did that a while back when I started this PR, and based on what I remember is I wanted to avoid issues where the prompt is an integer e.g., 

```
response = requests.post(
    "http://127.0.0.1:8000/predict", 
    json={"prompt": 4.0},
)
```

Does this casting to string cause problems somewhere? I am happy to undo it.